packages/vuetify/src/components/transitions/dialog-transition.tsx

```diff
@@ -35,11 +48,21 @@ export default defineComponent({
         })
         animation.finished.then(() => done())
       },
+      onAfterLeave (el: Element) {
+        (el as HTMLElement).style.removeProperty('pointer-events')
+      },
     }
 
     return () => {
```

Comment:
Could this be the following?

```tsx
return () => (
  <Transition
    name="dialog-transition"
    { ...(props.target ? functions : {}) }
    css={ !props.target }
    v-slots={ slots }
  />
)
```

Reply:
Probably but I think this is more readable.